clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/PartitionConsumptionState.java

```diff
@@ -603,10 +604,18 @@ public ChunkedValueManifest getRmdManifest() {
       return rmdManifest;
     }
 
+    public void setRmdManifest(ChunkedValueManifest rmdManifest) {
```

Comment:
Shall we remove the following params from the constructor since they are always wrong?
```
ChunkedValueManifest valueManifest,
      ChunkedValueManifest rmdManifest
```

Reply:
Yes, thanks for pointing out.  Will do in the next update.